src/Tribe/PUE/Checker.php

```diff
@@ -841,7 +841,15 @@ public function get_key( $type = 'any', $return_type = 'key' ) {
 				$class_name = $autoloader->get_prefix_by_slug( $this->get_slug() );
 
 				if ( $class_name ) {
-					$class_name .= 'PUE__Helper';
+					$is_namespaced = false !== strpos( $class_name, '\\' );
```

Comment:
This code assumes a change will be made to abstract https://github.com/moderntribe/tribe-common/pull/1349/files#diff-88be96e81f342137639526dddd8e8037R166-R171 logic further so that the prefixes data is already normalized for namespaces.